### PR TITLE
Create azure cloud provider connector

### PR DESCRIPTION
**Pull request by rathodmeetsatish**
Create azure cloud provider connector

### CODE REVIEW

**Comment by rathodmeetsatish** [comment]
trigger subcategoryfieldcheck

**Comment by rathodmeetsatish** [comment]
trigger subcategoryfieldcheck

File: internal/service/platform/connector/azure_cloud_provider_data_source.go [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+				},
+			},
+			"delegate_selectors": {
+				Description: "Connect using only the delegates which have these tags.",
+				Type:        schema.TypeSet,
+				Computed:    true,
+				Elem:        &schema.Schema{Type: schema.TypeString},
+			},
+			"azure_environment_type": {
+				Description: "Specifies the Azure Environment type, which is AZURE by default.Can either be AZURE or AZURE_US_GOVERNMENT",
```
Comment (rashminandasahoo) [inline internal/service/platform/connector/azure_cloud_provider_data_source.go:133]: Space missing after end period (default. Can).

File: internal/service/platform/connector/azure_cloud_provider_test.go [hunk 2]
```diff
@@ -0,0 +1,301 @@
...
+	}
+`, id, name)
+}
+
+func testAccResourceConnectorAzure_inheritFromDelegate_ua(id string, name string) string {
+	return fmt.Sprintf(`
+		resource "harness_platform_connector_azure_cloud_provider" "test" {
+			identifier = "%[1]s"
+			name = "%[2]s"
+			description = "test"
```
Comment (rashminandasahoo) [inline internal/service/platform/connector/azure_cloud_provider_test.go:259]: Let us replace "test" with "example" every where.

**Review by rashminandasahoo — CHANGES_REQUESTED** [review]

**Review by rashminandasahoo — APPROVED** [review]

**Pull request merged**